### DISCUSSION TITLE
Promoting the project. 🚀

### DISCUSSION THREAD

**Discussion by Spyware007** [Spyware007/Animating-Buttons#216] [General]
[**LinkedIn**](https://www.linkedin.com/posts/om-gawande_spyware007animating-buttons-activity-6991593574479859712-vfx2?utm_source=share&utm_medium=member_desktop)
[**Tweet**](https://twitter.com/oom_gawande/status/1585827768281202688)

**Like and Comment.** 👀

# Leave a 🌟 if you like it.

**Comment by ombhojane** [comment]
Do anyone get Runtime errors? I recently tried to fork and run, but it's not get run perfectly... Let me know it's happens only with me or not?
